Simbody/include/simbody/internal/GeneralForceSubsystem.h

```diff
@@ -85,7 +85,14 @@ class SimTK_SIMBODY_EXPORT GeneralForceSubsystem : public ForceSubsystem {
     be explicitly enabled. **/
     void setForceIsDisabled
        (State& state, ForceIndex index, bool shouldBeDisabled) const;
-
+       
+   /** Some of the dynamics calculations performed by the forces are
+     parallelized. Use this method to control the number of threads used when
+     calculating forces. The parallelism only affects forces that choose to
+     be executed in parallel. By default, the number of threads is the number
+     of cores on the machine. **/
+    void setNumberOfThreads(unsigned int numThreads);
```

Comment:
- use just `unsigned` rather than `unsigned int`
- should say "forces *may* be parallelized", and "only forces elements marked expensive are considered for parallel execution"
- should be clear about whether you really mean "number of cores" or "number of thread contexts"
- should have a matching `getNumberOfThreads()` to read how this is set
- should say when this method can be called; will it be ignored if called too late in the game? Or can I change the number of threads in the middle of execution?